schema.sql

```diff
@@ -1,10 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-id int,
+id INT,
 name VARCHAR(50),
 date_of_birth DATE,
 escape_attempts INT,
 neutered BOOL,
-weight_kg DECIMAL
+weight_kg DECIMAL,
+species VARCHAR(50)
 );
```

Comment:
- [ ] This will work if we were creating the tables for the first time, otherwise, we will need to DROP  the table first if it exists `DROP TABLE animals;` before adding it directly.  I think 🤔 A better way is to use `ALTER TABLE animals ADD species varchar(255);`

Reply:
OK that's what I did basically but I worte it like that in the file.

Reply:
Great 👍🏻 But you should add it too inside your schema file ✍🏻